src/NLog.Extensions.Logging/Logging/NLogLoggerProvider.cs

```diff
@@ -34,7 +40,18 @@ public NLogLoggerProvider()
         /// </summary>
         /// <param name="options"></param>
         public NLogLoggerProvider(NLogProviderOptions options)
+            :this(options, null)
+        {
+        }
+
+        /// <summary>
+        /// New provider with options
+        /// </summary>
+        /// <param name="options"></param>
+        /// <param name="logFactory">Optional isolated NLog LogFactory</param>
+        public NLogLoggerProvider(NLogProviderOptions options, LogFactory logFactory)
```

Comment:
Shouldn't this constructor be reachable via ConfigureExtensions ? (this constructor seems to never be called explicitly, only implicitly via ```public NLogLoggerProvider(NLogProviderOptions options) : this(options, null)```).

Reply:
I was thinking that people who needed to provide an isolated LogFactory was in a special unit-testing scenario, and didn't need any help from ConfigureExtensions